lib/build-es-document.js

```diff
@@ -46,17 +43,37 @@ const buildEsDocument = async ({ type, records }) => {
 
 const buildSierraBibs = (records) => {
```

Comment:
is this being called anywhere anymore?

Reply:
Good catch! Technically we don't need the reverse references any more, so it didn't break anything, but you're right I broke that wiring. Will fix.

Reply:
why don't we need the reverse references?

Reply:
Actually I was thinking of ES model back references, which are newly no longer needed because EsItem.aeonUrl no longer depends on anything in the parent EsBib. In the case of Sierra models, I'm not aware of any historical need, although i still think it's a sensible thing to have ready.

Reply:
Given that we don't actively need this behavior now, maybe we don't include it?

Reply:
If it wasn't already built and tested, we might opt not to build it, but as it is built and tested I would vote to include it. I also just think reverse model traversal is a sensible feature of a data model class like this. Maybe not essential enough to build it without an immediate need, but enough so to include it if it's already there.